tdp/core/variables/cluster_variables.py

```diff
@@ -52,9 +55,12 @@ def initialize_cluster_variables(
             (collection_name, collection.default_vars_directory.iterdir())
             for collection_name, collection in collections.items()
         ]
-        if override_folder:
+
+        for i, override_folder in enumerate(override_folders):
             override_folder = Path(override_folder)
-            collections_and_overrides.append(("overrides", override_folder.iterdir()))
+            collections_and_overrides.append(
+                (f"overrides_option_{i}", override_folder.iterdir())
```

Comment:
Out of curiosity, why did you use `overrides option_{i}` and not `overrides_{i}`. I don't really understand what the word "option" means.

Reply:
This was to reflect the fact that "--overrides" is a command line option, but I agree that's not the best idea as the paths can also come from the environment variable. We can call it "overrides_path_{i}" or just "overrides_{i}" as you suggested.

Reply:
The code in this function is not related to the CLI, for example, this function is called inside `tdp-lib` CLI and `tdp-server` too. `overrides_path_{i}` is great.

Reply:
Changed it to `overrides_path_{i}`. Do you want me to squash my 4 commits into one, or the merge will take care of it ?